app/experimenter/experiments/bugzilla.py

```diff
@@ -89,6 +98,24 @@ def create_experiment_bug(experiment):
         response_data = make_bugzilla_call(
             settings.BUGZILLA_CREATE_URL, bug_data
         )
+
+    # Firefox Version given might not be an available
+    # bugzilla tracking parameter, so remove and retry
+    err_code = response_data.get("code", None) == INVALID_PARAMETER_ERROR_CODE
+    if err_code:
+        tracking_msg = "cf_tracking_firefox" in response_data.get(
+            "message", None
+        )
+    if err_code and tracking_msg:
```

Comment:
You don't need to check err_code twice in a row, just embed this block inside the preceding if statement.